buildSrc/build.gradle

```diff
@@ -194,7 +194,14 @@ if (project != rootProject) {
   }
 
   String localDownloads = "${rootProject.buildDir}/local-downloads"
+  // TODO: keep a single copy once we have testclusters for example plugins
```

Comment:
I'm not sure I follow. Why do we need two copies of these?

Reply:
Some of the tests like the example plugins still use ClusterFormationTasks , but with this change, the `Version` class doesn't encode `-SNAPSHOT`. The result is that one will look for `8.0.0` and  the other for `8.0.0-SNAPSHOT` in the artifact name. This won't matter once we have everything on testclusters and we can clean it up then.